### ISSUE TITLE
update the github project with the latest version

### ISSUE THREAD

**Issue by loweryjk** [mdowds/python-mock-firestore#70]
I needed async support and spent a whole day implementing it. Just before I went to submit a merge request I found the MyPy page with a newer version containing an async implementation. So sad. :(

**Comment by benvdh-incentro** [comment]
@loweryjk Same here... we've been waiting for over a year, but @mdowds  activity seems to have stopped completely on GitHub since Q1 2022. 

You could try adding @anna-hope's fork directly as a python package by using the git URL (see second link):
https://github.com/anna-hope/python-mock-firestore

https://matiascodesal.com/blog/how-use-git-repository-pip-dependency/

Or use the python package by @nicoekkart:
https://pypi.org/project/mock-firestore-async/

**Comment by nicoekkart** [comment]
Yes, feel free to use my published version. Didn't include a reference there, but it is simply @anna-hope 's code with some small fixes on top of it.
